### DISCUSSION TITLE
How to prevent Popover from adding `padding-right` to the HTML style?

### DISCUSSION THREAD

**Discussion by s97712** [adobe/react-spectrum#6746] [Q&A]
The Popover Modal mode adds padding-right: 10px to the HTML, which is not always the intended behavior. However, I couldn’t find any way to prevent this behavior.

**Comment by yihuiliao** [comment]
I'm not sure if I'm seeing this. Do you have an example where you see this like a codesandbox or something similar?

**Comment by yihuiliao** [comment]
Following up on this since a team member let me know what you were experiencing. This is actually a known issue (https://github.com/adobe/react-spectrum/issues/1216, https://github.com/adobe/react-spectrum/issues/5309). There are some potential workarounds that might work for you but we haven’t had time to pick this up unfortunately.
